store/src/app/products/services/local-storage.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import { Product } from '../../interfaces/products.interface';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LocalStorageService {
+
+  constructor() {
+    this.checkLS
+  }
+
+  setToLS(products: Product[]): void {
```

Comment:
Better to use full name LocalStorage like `setToLocalStorage` or remove at all, like `setData`

Reply:
done